frontend/src/services/createPostService.test.js

```diff
@@ -20,7 +20,7 @@ describe('submitPost', () => {
       statusText: 'OK',
     });
     await submitPost('title', 'body');
-    expect(fetch.mock.calls[0][0]).toEqual('/api/post/create');
+    expect(fetch.mock.calls[0][0]).toEqual('/posts');
```

Comment:
I believe we still want the `/api` prefix. It will be fixed in backend soon (hopefully).
We might do a sweep to fix all `/api` urls when backend is ready.

Reply:
(So probably we could leave these for now so things still work)

Reply:
(So probably we could leave these for now so things still work)

Reply:
I was thinking leave it for now as thats whats funcitonal, but create new issue for when backend is fully completed

Reply:
Sounds like a good idea.